### PR TITLE
colab edits and show cell outputs

### PR DESCRIPTION
**Pull request by ezekielemerson**
- edited data rows colab to include all the data types we offered
- displayed the required tool names in a table in the ontologies colab
- displayed appropriate outputs for the text and video export colabs I uploaded earlier this week

### CODE REVIEW

File: examples/basics/ontologies.ipynb [hunk 1]
```diff
@@ -85,8 +85,18 @@
             "cell_type": "markdown",
```
Comment (jtsodapop) [inline examples/basics/ontologies.ipynb:85]: <p>Do we believe this is going to be particularly useful for customers? Are we finding confusion from customers at this moment about this?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

File: examples/label_export/text.ipynb [hunk 2]
```diff
@@ -2,22 +2,17 @@
   "cells": [
```
Comment (jtsodapop) [inline examples/label_export/text.ipynb:2]: <p>do we want to show this output?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

File: examples/label_export/video.ipynb [hunk 3]
```diff
@@ -2,22 +2,17 @@
   "cells": [
```
Comment (jtsodapop) [inline examples/label_export/video.ipynb:2]: <p>is it helpful to show this, or can we leave it up to the user to run the code block to see it themselves? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/537/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

File: examples/basics/ontologies.ipynb (hunk 1 above)
Comment (ezekielemerson) [reply examples/basics/ontologies.ipynb:85]: Yes, I have seen a few customers trying this out and not knowing the names for certain tools that you must use, and it's not well documented.

**Review by jtsodapop — APPROVED** [review]

**Pull request merged**